homeassistant/components/onewire/entity.py

```diff
@@ -54,7 +54,6 @@ def extra_state_attributes(self) -> dict[str, Any] | None:
         """Return the state attributes of the entity."""
         return {
             "device_file": self._device_file,
-            "raw_value": self._value_raw,
```

Comment:
Unless we need to deprecate it?

Reply:
I believe we deprecate them, which for attributes is just announcing that it will be removed later and a comment in the code.

Also, what is devicefile?

Reply:
device_file is the full path of the underlying property on the 1-wire bus.

eg. `/12.111111111111/sensed.A` means `sensed.A` on root device `12.111111111111`

eg. `/1F.111111111111/main/1D.111111111111/counter.B` means `counter.B` on device `1D.111111111111` from branch `main` on root device `1F.111111111111`

It is a kind of static diagnostic entity property